### ISSUE TITLE
Add `GeoInterface.convert` methods

### ISSUE THREAD

**Issue by rafaqz** [evetion/WellKnownGeometry.jl#18]
Would be nice if `convert` worked ;)

We can add a `convert(WellKnownGeometry, geom)` method now too.

**Comment by evetion** [comment]
Yeah, first pirating GeoFormattypes? Not sure if that clashes with ArchGDAL.

Will make type wrappers that also store dimensionality.

**Comment by rafaqz** [comment]
It wouldn't be pirating in the module method. But yes maybe ArchGDAL should do the convert to GeoFormatTypes wrappers.
